test/unit/css.js

```diff
@@ -664,6 +654,190 @@ QUnit.test( "show() after hide() should always set display to initial value (#14
 	assert.equal( div.css( "display" ), "list-item", "should get last set display value" );
 } );
 
+QUnit.test( "show/hide 3.0, default display", function( assert ) {
```

Comment:
These tests looks pretty complicated, i would suggest following some sort of [AAA](http://c2.com/cgi/wiki?ArrangeActAssert) strategy.

This could be done latter, but i would highly encourage you do it